packages/chat-stateful-client/package.json

```diff
@@ -38,11 +38,11 @@
     "nanoid": "3.3.6"
   },
   "peerDependencies": {
-    "@azure/communication-chat": "1.6.0-beta.1 || >=1.5.0"
+    "@azure/communication-chat": "1.6.0-beta.2 || >=1.5.0"
   },
   "devDependencies": {
-    "@azure/communication-chat": "1.6.0-beta.1 || ^1.5.0",
-    "@azure/communication-signaling": "1.0.0-beta.23 || 1.0.0-beta.26",
+    "@azure/communication-chat": "1.6.0-beta.2 || ^1.5.0",
+    "@azure/communication-signaling": "1.0.0-beta.27 || 1.0.0-beta.26",
```

Comment:
Can we remove this signaling dependency as it should be part of the chat deps?

Also, why are these dependencies under devDependencies?

Reply:
We directly call to `"@azure/communication-signaling"` in the chat stateful test so probably why they are just in dev dependencies.

Reply:
The chat dependency can go though, right?